src/dashboard/cd-dashboard-projects.vue

```diff
@@ -41,7 +41,11 @@
       userLocale() {
         const langCookie = LocaleService.getUserLocale();
         if (langCookie) {
-          return langCookie.replace(/"/g, '').replace('_', '-');
+          let formattedLocale = langCookie.replace(/"/g, '').replace('_', '-');
+          if (/en-.*/.test(formattedLocale)) {
```

Comment:
I reckon this would happen for other locales, like es-ES, es-AR, es-MX, etc. Worth checking, and if so probably makes more sense to just pull out the first two characters, or split by the `-` (or the `_` and don’t do the replacement...)

Reply:
Yes good point.
Will check on that.

Reply:
I guess there's a question about whether this is needed anymore. It'll result in one less request, and it's unlikely we're gonna differentiate between en-US and en-GB, so I guess keep it..?

Reply:
I was thinking that as well, it feels like it's probably worth it to save a request so will leave it in.

(Fast forward to bug in three years time when en-GB and en-US are added to projects api)